packages/harvester/harvester-csi-driver/generated-changes/patch/Chart.yaml.patch

```diff
@@ -10,7 +10,8 @@
    catalog.cattle.io/os: linux
 -  catalog.cattle.io/permits-os: linux
 +  catalog.cattle.io/permits-os: linux,windows
-   catalog.cattle.io/rancher-version: '>= 2.6.1-0 < 2.7.0-0'
+-  catalog.cattle.io/rancher-version: '>= 2.6.1-0 < 2.7.0-0'
++  catalog.cattle.io/rancher-version: '>= 2.6.1-0 < 2.8.0-0'
```

Comment:
should be >= 2.7.0-0 < 2.8.0-0, a 101 chart would never be tested in 2.6

Reply:
> should be >= 2.7.0-0 < 2.8.0-0, a 101 chart would never be tested in 2.6

How to be compatible going forward >= 2.6.0-0 < 2.7.0-0